docs/effects/README.md

```diff
@@ -99,8 +99,26 @@ import { AdminEffects } from './effects/admin';
 })
 export class AdminModule {}
 ```
+## Effects that use State
```

Comment:
This should go as a section in the API. Generally, you should avoid selecting state in your effects, but instead provide any data the effect needs as part of the action payload.